admin/views/class-yoast-integration-toggles.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class for managing integration toggles.
+ */
+class Yoast_Integration_Toggles {
+
+	/**
+	 * Available integration toggles.
+	 *
+	 * @var array
+	 */
+	protected $toggles;
+
+	/**
+	 * Instance holder.
+	 *
+	 * @var self|null
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Gets the main integration toggles manager instance used.
+	 *
+	 * This essentially works like a Singleton, but for its drawbacks does not restrict
+	 * instantiation otherwise.
+	 *
+	 * @return self Main instance.
+	 */
+	public static function instance() {
+		if ( self::$instance === null ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Gets all available integration toggles.
+	 *
+	 * @return array List of sorted Yoast_Feature_Toggle instances.
```

Comment:
Yes, I missed that.

Reply:
I deleted my previous comments because I looked into it better.
Actually, let me recap:
* the features tab uses a `Yoast_Feature_Toggles` class who manages an array of `Yoast_Feature_Toggle` objects
* the integration tab uses a `Yoast_Integration_Toggles` class who manages  an array of `Yoast_Feature_Toggle` objects as well.
This is because the `Yoast_Feature_Toggle` has everything we need for our toggle, so creating a `Yoast_Integration_Toggle` (by copying or by subclassing) seemed unnecessary to me...

So if we stick to using the `Yoast_Feature_Toggle` class the doc comment is right.